ports/cpp-jsonlib/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "cpp-jsonlib",
```

```suggestion
  "name": "mysvac-cpp-jsonlib",
```

After removing the common prefix (`cpp`) and suffix (`lib`), we are left with just `json` as the stem. Which may be confused with unrelated projects `jsonccp` and `jsonlib`. The name `cpp-jsonlib` does not conform to our [port naming guidelines](https://learn.microsoft.com/en-us/vcpkg/contributing/maintainer-guide#check-names-against-other-repositories). An easy solution is to prefix the project owner to the port name (`mysvac`).

---

OK，changed.